Finance/teller-track-spending.1h.sh

```diff
@@ -7,7 +7,7 @@
 # - teller-cli: https://github.com/sebinsua/teller-cli#from-release
 # - pcregrep: `brew install pcre`
 #
-# <bitbar.title>teller-track-spending</bitbar.title>
+# <bitbar.title>Track spending at UK banks</bitbar.title>
```

Comment:
Can I put a hyperlink in the description actually? Might be a good idea to draw attention to signup.